src/customConfigTests/src/test/resources/hoverfly/github-oauth2-login.json

```diff
@@ -49,13 +49,17 @@
             "matcher" : "exact",
             "value" : "gzip, deflate, br"
           } ],
+          "Accept-Language" : [ {
+            "matcher" : "exact",
+            "value" : "en-US"
+          } ],
           "Connection" : [ {
             "matcher" : "exact",
             "value" : "keep-alive"
           } ],
           "Referer" : [ {
             "matcher" : "glob",
-            "value" : "http://localhost:8888/weblogin*"
+            "value" : "http://localhost:8888/*"
```

Comment:
No idea why this behaviour has changed during Github Oauth authentication - chromedriver change?